apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -374,7 +399,16 @@ void _wt_scan_done(wifi_manager_scan_info_s **scan_result, wifi_manager_scan_res
 		WT_TEST_SIGNAL;
 		return;
 	}
-	_wt_print_scanlist(*scan_result);
+	if (g_scan_join == 0) {
+		_wt_print_scanlist(*scan_result);
+	} else {
+		// request type is WT_TYPE_SJOIN. so it doesn't print scan list
+		// and pass scan list result to _wt_scan_connect;
+		g_scanned_result = _wt_get_scanned_list(*scan_result,
+												g_scanned_ssid,
```

Comment:
How about removing unnecessary tabs for readability?

Reply:
it's dependent to what ide do you use. So I don't want to modify it.